homeassistant/components/shelly/binary_sensor.py

```diff
@@ -190,6 +190,13 @@ class RestBinarySensorDescription(RestEntityDescription, BinarySensorEntityDescr
         entity_category=EntityCategory.DIAGNOSTIC,
         supported=lambda status: status.get("apower") is not None,
     ),
+    "smoke": RpcBinarySensorDescription(
+        key="smoke:0",
```

Comment:
I think it should work without the `:0` which will be more future proof if there is a device with multiple sensors or a different sensor ID
```suggestion
        key="smoke",
```

Reply:
Yeah, good point.